tests/test_room_resource/test_delete_resource.py

```diff
@@ -32,3 +32,10 @@ def test_non_existant_deleteresource_mutation(self):
             delete_non_existant_resource,
             "Resource not found"
         )
+
+    def test_deleteresource_in_room_that_is_not_in_admin_location(self):
```

Comment:
@bevkololi I think you meant to name it `test_delete_resource_in_room_that_is_not_in_admin_location`.
Also checkout line `14` and `35` in `test_update_room_resource.py` for the same naming issue

Reply:
Thanks @dnuwa, I've updated this.